src/backend/src/utils/work-packages.utils.ts

```diff
@@ -38,6 +38,30 @@ export const createChangeJsonNonList = (
   return undefined;
 };
 
+// create a change json if the old and new value are different, otherwise return undefined
+export const createChange = (nameOfField: string, oldValue: any, newValue: any, crId: number, implementerId: number) => {
+  if (oldValue == null) {
```

Comment:
what if new value is null too? (the first line should probs be if oldvalue===newValue return undefined)